### ISSUE TITLE
Allow search for BCPs with space in name

### ISSUE THREAD

**Issue by jimfenton** [ietf-tools/datatracker#4328]
### Description

Currently, a search on Datatracker for a BCP fails: nothing is found for "BCP 195" although a search for "BCP195" does return the intended document (RFC 7525). 

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by rjsparks** [comment]
The datatracker doesn't really understand BCPs right now - it tries to use DocAlias to point to a Document of type 'draft' that happens to be an RFC, and it can only point to one, so trying to do anything with any BCP with more than one RFC in it simply won't work.

What you ask will need to wait until after we change the models to make RFCs their own document type, rework DocAlias completely, and then (I currently plan) to add BCP and STD document types that consist of nothing but DocumentRelations to the constituent RFCs.

In other words, this is a small aspect of a _much_ bigger problem, and simply trying to change the search to be space resilient won't even really solve this one.

**Comment by larseggert** [comment]
FWIW, this works in the top-right search box.

**Comment by rjsparks** [comment]
Well, no it doesn't. Typing`BCP 9` there finds stuff, but not BCP9. `BCP9` provides a link for BCP9, but if you click that, you only get RFC6410, not the set of RFCs that make up BCP9. This really isn't going to be fixed until we do the work to make RFCs (and subseries) their own document type. I'm hoping to get to do that post 116 (post postgres and django4).

**Comment by rjsparks** [comment]
Fixed at 4b1274c44d011f55578cd7f35d05d1e5e1a09357 on feat/rfc. Closing this, but note that it will be some time before this is released into production. In the meantime, you can exercise the result at dt-rfc.dev.ietf.org.
